src/locales/zh/main.json

```diff
@@ -146,6 +146,37 @@
     "resetView": "重置视图",
     "selectMode": "选择模式",
     "toggleLinkVisibility": "切换连线可见性",
+    "Inputs": "输入",
```

Comment:
Please add the change to `en/main.json`. The github action will auto generate translations of other locales.